Main.py

```diff
@@ -1,82 +1,102 @@
+                if lag:
                     lagging = (simEnv['lag_time'][0:nr_alive] > 0) #.dot(simEnv['age'][0:nr_alive] >= 1)
                     lag = any(lagging)
                     simEnv['lag_time'][lagging] -= MINIMUM_TIME_RESOLUTION
                     simEnv['next_divition'][lagging] += MINIMUM_TIME_RESOLUTION
-                    print lagging
 
                 ## Divide
                 toDivide = (simEnv['next_divition'][0:nr_alive] <= t)#.dot(simEnv['lag_time'][0:nr_alive] >= t)
                 nr_Divide = np.sum(toDivide)
                 if nr_Divide != 0:
-                    #print nr_Divide
-                    #print np.nonzero(toDivide)[0]
+                    # print nr_Divide
+                    # print np.nonzero(toDivide)[0]
                     idxDivide = np.nonzero(toDivide)[0]
                     toBirth = np.arange(nr_alive,(nr_alive + nr_Divide))
                     simEnv['founder_id'][toBirth] = idxDivide
                     simEnv['lag_time'][toBirth] = simEnv['lag_time'][toDivide]
                     simEnv['cell_cycle_time'][toBirth] = simEnv['cell_cycle_time'][toDivide]
                     simEnv['next_divition'][toDivide] = t + simEnv['cell_cycle_time'][toDivide]
                     simEnv['next_divition'][toBirth] = simEnv['next_divition'][toDivide]
-                    #CELL_CYCLE_TIMES[simEnv['founder_id'][toDivide]]
+                    # CELL_CYCLE_TIMES[simEnv['founder_id'][toDivide]]
                     simEnv['nr_divitions'][toDivide] += 1
 
                     ## Mutate
                     toMutate = np.random.random(nr_Divide) < PROB_MUTATION
                     if any(toMutate):
-                        mutateORF = np.random.random([sum(toMutate),1]) * (data1_Cum[-1]) # pick mutation
+                        mutateORF = np.random.random([sum(toMutate), 1]) * (data1_Cum[-1]) # pick mutation
                         idxORF = np.searchsorted(data1_Cum, mutateORF, side='right') # peek mutation
                         mutation[toBirth[toMutate]] = idxORF  # store mutation
-                        #idxDivide = np.nonzero(toDivide)[0]
-                        #nr_mutations +=
+                        # idxDivide = np.nonzero(toDivide)[0]
+                        # nr_mutations +=
                         print 'time is ', t
                         print 'alive: ', nr_alive
-                        print data4[idxORF,iEnv]
-
+                        print data4[idxORF, iEnv]
                         simEnv['cell_cycle_time'][toBirth[toMutate]] = simEnv['cell_cycle_time'][idxDivide[toMutate]] + \
-                                                                       data4[idxORF,iEnv]*MEAN_CELL_CYCLE_TIME #(CELL_CYCLE_TIMES[simEnv['founder_id'][idxDivide[toMutate]]])
+                                                                       data4[idxORF, iEnv]*MEAN_CELL_CYCLE_TIME #(CELL_CYCLE_TIMES[simEnv['founder_id'][idxDivide[toMutate]]])
```

Comment:
Consider 

```
simEnv = \
    simEnv + data4 * MEAN_CELL...

```

for readability.
Also please update `data4` to have a descriptive name.